<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center3D_Dialog</class>
 <widget class="QDialog" name="ui_DD_tacker_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>186</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>DD_Tacker - matchmove</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>429</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupCam">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Camera</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="listCam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupStat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>300</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QLabel" name="enbLoc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>select Camera and one Locator</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnEnable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDisable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
